package.json

```diff
@@ -21,10 +21,12 @@
     "express-session": "^1.15.6",
     "hashids": "^1.1.4",
     "ip": "^1.1.4",
+    "jsonwebtoken": "^8.1.1",
     "minio": "^3.2.3",
     "multer": "^1.3.0",
     "nconf": "^0.8.5",
     "request": "^2.83.0",
+    "rsa-pem-from-mod-exp": "^0.8.4",
```

Comment:
Check on npm and make sure its a well used and maintained package.

Reply:
Last updated 4 years ago. No docs.

Reply:
I checked the source on GitHub looks fine. Its also getting good traction on NPM so its in use.